docs/04_web/04_integrations/02_Angular/03_sharing-data.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Angular - Sharing data'
+sidebar_label: 'Sharing data'
+id: angular-sharing-data
+keywords: [web, integrations, angular, sharing-data]
+tags:
+    - web
+    - integrations
+    - angular
+---
+This section of the documentation explores how to pass data from an Angular application to web components. Understanding the nuances of sharing data is crucial for seamless integration and functionality. We will discuss how to pass both primitive and non-primitive values.
+
+## Passing Values (Primitive and Non-Primitive)
+
+In Angular, both primitive values (such as strings, numbers, and boolean values) and non-primitive values (like objects and arrays) can be passed directly as attributes from Angular to web components, similar to how data is passed to Angular components.
+
+### Primitive Values
+For primitive values like strings, numbers, and boolean values (`true`/`false`), you can bind them in the HTML template like so:

```

Comment:
should we add a link here to angular docs for passing props.

For complete information please see the angular documenation?

https://angular.io/guide/inputs-outputs

Reply:
we have nothing on `Output` `EventEmitter` -  do we need that here?
 
It is comprehensively explained on the angular docs though